compair/api/group.py

```diff
@@ -44,6 +44,16 @@ def post(self, course_uuid):
         params = new_group_parser.parse_args()
 
         new_group.name = params.get("name")
+
+        # check if group name is unique
+        group_name_exists = Group.query \
```

Comment:
I'd like the unique constraint added on course_id and name instead
also this check didn't include the `course_id` check

Reply:
Sorry what do you mean instead? If we add that constraint, we won't need this check here anymore?